docs/changelog/index.md

```diff
@@ -11,6 +11,12 @@ an action on your part will be marked "_action optional_".
 
 ## Changelog
 
-### **2018-06-11** [Added encrypted temporary directory](0001-add-encrypted-tmp.md) (_action required_)
+### **2018-07-13** [Changelog](0002-supervisor-service-definitions.md) (_action required_) 
+
+There are several CommCare specific processes that are defined in supervisorconfiguration files. This change decouples the process definitions from code.
+
+### **2018-06-11** [Added encrypted temporary directory](0001-add-encrypted-tmp.md) (_action required_) 
```

Comment:
Nit: Trailing single space on this line (which is what creates the diff here)

Reply:
There shouldn't be a space below the header? There is one in the file I was going off of (line 15 here): https://github.com/dimagi/commcare-cloud/pull/1927/files#diff-8126fa4ec84c1259fc7a6ea789a1ac53

Reply:
The space I'm talking about is _in_ this line, the last character before the newline.

Reply:
ooooh I was interpreting your comment as a trailing newline. got it.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1979/commits/a516bc3342f8e0bded4bfa2bb1e07dba8169d99b